examples/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "handsontable-examples",
+  "private": true,
+  "scripts": {
+    "clean": "node ./scripts/clean-subpackages.mjs",
+    "test": "node ../scripts/code-examples.mjs test next",
+    "build": "node ../scripts/code-examples.mjs build next",
```

the script location changed, but the paths didn't

```suggestion
    "test": "node ./scripts/code-examples.mjs test next",
    "build": "node ./scripts/code-examples.mjs build next",
```